### PR TITLE
Rewrite the entire code

### PR DESCRIPTION
**Pull request by Stilic**
I have done this rewrite because most of the code is outdated and is using for example old versions of Bootstrap and Angular.

### CODE REVIEW

**Review by Dabbyboigamer — APPROVED** [review]

File: pages/about.html [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+    <p class="text-justify">
+        An <em><strong>enormous</strong></em> thank you to the original
+        developers of
+        <a href="https://www.aseprite.org" rel="nofollow">Aseprite</a>, without
+        them and their original licensing this project wouldn’t exist.
+        Additional shout-outs are given in the
+        <a
+            href="https://github.com/LibreSprite/LibreSprite/blob/master/CONTRIBUTORS.md"
+            >list of contributors</a
+        >. If you have submitted work to LibreSprite and would like to have your
```
Comment (KTibow) [inline pages/about.html:28]: what the heck is this formatting

Comment (Stilic) [reply pages/about.html:28]: it is the formatting done by Prettier (with the default preset).

Comment (Stilic) [reply pages/about.html:28]: maybe i can fix that by using a `.prettierrc` file.

Comment (KTibow) [reply pages/about.html:28]: check the whitespace sensitivity option

**Pull request closed without merging**